aspnetcore/signalr/authn-and-authz.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Authentication and Authorization in SignalR
+author: rachelappel
+description: Learn how to use authentication and authorization in ASP.NET Core SignalR.
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 06/29/2018
+uid: signalr/authn-and-authz
+---
+
+# Authentication and authorization in SignalR
+
+By [Andrew Stanton-Nurse](https://twitter.com/anurse)
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/signalr/authn-and-authz/sample/) [(how to download)](xref:tutorials/index#how-to-download-a-sample)
+
+## Authenticate users connecting to a SignalR hub
+
+SignalR can be used with [ASP.NET Core Authentication](xref:security/authentication/index) to associate a User with each connection. In a hub, authentication data can be accessed from the [`HubConnectionContext.User`](/dotnet/api/microsoft.aspnetcore.signalr.hubconnectioncontext.user?view=aspnetcore-2.1) property. Authentication also allows the hub to call methods on all connections associated with a user (See [Manage users and groups in SignalR](xref:signalr/groups) for more information). Multiple connections may be associated with a single user.
+
+### Cookie Authentication
+
+In a browser-based app, cookie authentication allows your existing user credentials to automatically flow to SignalR connections. When using the browser client, no additional configuration is needed. If the user is logged in to your app, the SignalR connection automatically inherits this authentication.
+
+We do not recommend using Cookie authentication unless you only need to authenticate users from the browser client. When using the .NET Client, the `Cookies` property can be configured in the `.WithUrl` call in order to provide a cookie. However, this requires that you provide an API to exchange authentication data for a cookie.
```

Comment:
- do not --> don't
- link the ".NET Client" text to the associated doc (xref:signalr/dotnet-client)